src/main.cpp

```diff
@@ -852,6 +877,12 @@ bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransa
                                    hash.ToString(), nSigOps, MAX_STANDARD_TX_SIGOPS),
                              REJECT_NONSTANDARD, "bad-txns-too-many-sigops");
 
+        // Expire old transactions before trying to replace low-priority ones.
+        int expired = pool.Expire(GetTime() - GetArg("-mempoolexpiry", DEFAULT_MEMPOOL_EXPIRY) * 60 * 60);
```

Comment:
Can't ```- GetArg("-mempoolexpiry", DEFAULT_MEMPOOL_EXPIRY) * 60 * 60``` be inside pool.Expire?, it may become an attribute of the txmempool in the future or something.